src/validation/Validator.ts

```diff
@@ -530,8 +530,8 @@ export class Validator {
      * Checks if the string is numeric.
      * If given value is not a string, then it returns false.
      */
-    isNumberString(value: unknown, options?: ValidatorJS.IsNumericOptions): boolean {
-        return typeof value === "string" && this.validatorJs.isNumeric(value, options);
+    isNumberString(value: unknown, options?: IsNumberOptions): boolean {
+        return typeof value === "string" && this.validatorJs.isNumeric(value) && (options == null || this.isNumber(parseFloat(value), options));
```

Comment:
It is necessary ` && (options == null || this.isNumber(parseFloat(value), options)` part? I think `isNumber` accept also undefined as option argument.